app/Http/Repository/UserRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Suyabay\Http\Repository;
+
+use Suyabay\User;
+
+class UserRepository
+{
+
+    /**
+    * Return all user from the database
+    */
+	public function getAllUsers()
+	{
+		return User::all();
```

Comment:
Where is `return User::all()` running to?..Why is there so much space between where the opening brace is and `return User::all()` is?